packages/caravan-fees/src/tests/btcTransactionComponents.test.ts

```diff
@@ -180,7 +180,7 @@ describe("BtcTxOutputTemplate", () => {
       });
       expect(() => {
         output.lock();
-      }).toThrow("Output is already locked and cannot be modified.");
```

Comment:
maybe just add that you expect it to remain locked as well as not throw.

Reply:
corrected [here](https://github.com/caravan-bitcoin/caravan/pull/114/commits/c75270a9cbe9684263429ec2711698486c77858f#diff-1071cfaded8c65c06312b83fa57a44238eec620c9dcc699cf358f3301bcc4458R179) 

and added more tests for rbf and validation methods